examples/replication/main.tf

```diff
@@ -15,37 +15,41 @@ module "resource_group" {
 
 module "cos_source_bucket" {
-  source                    = "../../"
-  bucket_name               = "${var.prefix}-bucket-source"
-  resource_group_id         = module.resource_group.resource_group_id
-  region                    = var.region
-  cos_instance_name         = "${var.prefix}-source-cos"
-  cos_tags                  = var.resource_tags
-  access_tags               = var.access_tags
-  object_versioning_enabled = true
-  kms_encryption_enabled    = false
-  retention_enabled         = false
-  archive_days              = null
-  expire_days               = null
+  source                             = "../../"
+  bucket_name                        = "${var.prefix}-bucket-source"
+  resource_group_id                  = module.resource_group.resource_group_id
+  region                             = var.region
+  cos_instance_name                  = "${var.prefix}-source-cos"
+  cos_tags                           = var.resource_tags
+  access_tags                        = var.access_tags
+  object_versioning_enabled          = true
+  kms_encryption_enabled             = false
+  retention_enabled                  = false
+  archive_days                       = null
+  expire_days                        = null
+  activity_tracker_read_data_events  = false # disable activity_tracker
+  activity_tracker_write_data_events = false # disable activity_tracker
 }
 
 ##############################################################################
 # Create COS target bucket
 ##############################################################################
 
 module "cos_target_bucket" {
-  source                    = "../../"
-  bucket_name               = "${var.prefix}-bucket-target"
-  resource_group_id         = module.resource_group.resource_group_id
-  region                    = var.region
-  cos_instance_name         = "${var.prefix}-target-cos"
-  cos_tags                  = var.resource_tags
-  access_tags               = var.access_tags
-  object_versioning_enabled = true
-  kms_encryption_enabled    = false
-  retention_enabled         = false
-  archive_days              = null
-  expire_days               = null
+  source                             = "../../"
+  bucket_name                        = "${var.prefix}-bucket-target"
+  resource_group_id                  = module.resource_group.resource_group_id
+  region                             = var.region
+  cos_instance_name                  = "${var.prefix}-target-cos"
+  cos_tags                           = var.resource_tags
+  access_tags                        = var.access_tags
+  object_versioning_enabled          = true
+  kms_encryption_enabled             = false
+  retention_enabled                  = false
+  archive_days                       = null
+  expire_days                        = null
+  activity_tracker_read_data_events  = false # disable activity_tracker
+  activity_tracker_write_data_events = false # disable activity_tracker
```

Comment:
There is no need to set these to false - just remove them and allow the default to be used